app/TracingStrategyAssets.js

```diff
@@ -0,0 +1,105 @@
+import { useTranslation } from 'react-i18next';
+import { Images } from './assets';
+import { config } from './COVIDSafePathsConfig';
+
+export function AppSpecificAssets() {
```

Comment:
This is basically a hook now, so call it useAssets or similar.

Knowing it's a hook means it's important that it's always run in a predictable order.

Reply:
Thanks - will do!